app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    action, source_file, destination_file = command.split()
+
+    if action != "cp" or len(command.split()) != 3:
+        raise ValueError("Command in wrong format")
+    if source_file != destination_file:
+        try:
+            with (
+                open(source_file, "r") as file_in,
+                open(destination_file, "w") as file_out
+            ):
+                file_out.write(file_in.read())
+        except FileNotFoundError:
```

Comment:
OSError would be more appropriate exception here - it will cover all file-related errors